schemas/EGA.individual.json

```diff
@@ -0,0 +1,72 @@
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "$id": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.individual.json",
+    "type": "object",
+    "title": "EGA individual metadata schema v0.0.1",
+    "description": "Metadata schema used by the European Genome-phenome Archive (EGA) to validate its individual metadata object. This object is intended to contain metadata about the individual, also known as the sample donor or subject of study. An individual is defined as a person that participates in an experiment or from which a material sample was derived. Further details can be found in the EGA-metadata-schema GitHub repository (https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas) and EGA-archive website (https://ega-archive.org/)",
```

Comment:
```suggestion
    "description": "Metadata schema used by the European Genome-phenome Archive (EGA) to validate its individual metadata object. This object is intended to contain metadata about the individual, also known as the sample donor or subject of study. An individual is defined as a person that participates in an experiment and from which a material sample was derived. Further details can be found in the EGA-metadata-schema GitHub repository (https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas) and EGA-archive website (https://ega-archive.org/)",
```

Reply:
I'm not entirely sure about this suggestion, since I did put ``or`` on purpose. If it is phrased with ``and``, it implies that all individuals had some physical sample derived from them. Given the spectra of analyses we contain, I thought that some individuals may be associated to analyses where no biomaterial was extracted. 

If that is not the case and we **always** have derived physical samples from all individuals, I would change it as you suggest.